src/applications/vaos/components/AddToCalendar.jsx

```diff
@@ -152,13 +154,23 @@ export default function AddToCalendar({
   }
 
   return (
-    <a
+    <va-link
+      download
```

Comment:
No need for 'download' property since the 'calendar' property is the one that displays the calendar icon

Reply:
@jenniemc Hah! Good thing I was the one that added that functionality to the actual web component. Totally missed that.